Cargo.toml

```diff
@@ -59,20 +59,26 @@ glsl-to-spirv = { version = "0.1", optional = true }
 log = "0.4"
 rand = "0.8"
 rayon = "1"
-ron = "0.6"
+ron = "0.5.1"
```

Comment:
why are we downgrading ron?

Reply:
Because of this issue: https://github.com/ron-rs/ron/issues/335 that was fixed but not yet released.

Reply:
filed https://github.com/ron-rs/ron/issues/365

Reply:
Can we use 0.7.1 now?

Reply:
Think so. Btw, I think need to test again emscripten integration. So I close it for now.